assets/questions/pirateTranslator.js

```diff
@@ -224,6 +273,61 @@ globalData.questions['pirateTranslator'] = {  // eslint-disable-line dot-notatio
           "Don't forget spaces in your output."
         ].join('')
       ]
+    }, {
+      buggyFunctionName: 'AuxiliaryCode.alwaysPrependSpace',
+      ignoredTestSuiteIds: [],
+      messages: [
+        [
+          'Spaces should only appear between words, not at the ',
+          'beginning of the sentence.'
+        ].join('')
+      ]
+    }, {
+      buggyFunctionName: 'AuxiliaryCode.alwaysAppendSpace',
+      ignoredTestSuiteIds: [],
+      messages: [
+        [
+          'Spaces should only appear between words, not at the ',
+          'end of the sentence.'
+        ].join('')
+      ]
+    }, {
+      buggyFunctionName: 'AuxiliaryCode.onlySpaceAfterPirateWords',
+      ignoredTestSuiteIds: [],
+      messages: [
+        [
+          'Spaces should only appear between each whether English ',
```

Comment:
This is unclear, I think. Perhaps: "Your output seems to be missing some spaces." followed by "All words should be separated by spaces, regardless of whether they are English words or Pirate words."